internal/db/seeds/dev_small.sql

```diff
@@ -8,7 +8,8 @@ INSERT INTO accounts(id, account_number, org_id)
 VALUES (1, '13', '000013'),
        (2, '15', '000015'),
        (3, NULL, '000042'),
-       (4, NULL, '000077')
+       (4, NULL, '000077'),
+       (5, '6089719', '000016')
```

Comment:
You need to make sure you have this org_id linked to this account number in sources. 
(It is there on default) @ezr-ondrej

Reply:
Sure makes sense, but not all our endpoints will interact with sources. Our own accounts still make sense.